services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_reports/diagnostics/recommendations.tsx

```diff
@@ -422,6 +459,23 @@ export const Recommendations = ({ passedPreviouslyAssignedRecommendations, passe
     })
   }
 
+  function assignPostTest() {
+    const dataToPass = {
+      classroom_id: classroomId,
+      student_ids: postTestSelections,
+      unit_template_id: postDiagnosticUnitTemplateId
+    }
+    setPostTestAssigning(true)
+    requestPost('/teachers/progress_reports/assign_post_test/', dataToPass, (data) => {
+      setPostTestAssigning(false)
+      setPostTestAssigned(true)
+      getPreviouslyAssignedPostTestData()
+    }, (data) => {
+      alert('We had trouble processing your request. Please check your network connection and try again.');
```

Comment:
I think since we use this same message in a few other instances in this file, we should pull it out into a shared const